src/librustc_mir/interpret/eval_context.rs

```diff
@@ -768,7 +768,10 @@ impl<'mir, 'tcx, M: Machine<'mir, 'tcx>> InterpCx<'mir, 'tcx, M> {
         // FIXME(oli-obk): make this check an assertion that it's not a static here
         // FIXME(RalfJ, oli-obk): document that `Place::Static` can never be anything but a static
         // and `ConstValue::Unevaluated` can never be a static
-        let param_env = if self.tcx.is_static(gid.instance.def_id()) {
+        // FIXME(oli-obk, spastorino): the above FIXME is not true anymore, PlaceBase::Static does
+        // not exist anymore (except for promoteds but it's going away soon).
```

Comment:
Fixed

Reply:
Done

Reply:
It doesn't look like it is.

Reply:
Ohh that's part of a commit Oli made which should go away anyway I'm going to fix with a last commit and in case Oli rebases things and fix that in his PR I can just remove it.

Reply:
Fixed